raiden/console.py

```diff
@@ -214,25 +213,23 @@ def register_asset(self, token_address):
         self._raiden.register_channel_manager(channel_manager)
         return channel_manager
 
-    def ping(self, peer):
+    def ping(self, peer, timeout=0):
         """See, if a peer is discoverable and up.
```

Comment:
@hackaugusto I will undo my docstring change here. There is no reason to introduce new documentation style if sphinx can work with the napoleon syntax. Thanks for prompting me to look that up.